packages/client/mutations/toasts/mapRequestToJoinOrgToToast.ts

```diff
@@ -36,7 +37,7 @@ const mapRequestToJoinOrgToToast = (
     secondaryAction: {
       label: 'Deny',
       callback: () => {
-        // Do nothing
+        SendClientSegmentEventMutation(atmosphere, 'Join Request Denied')
```

Comment:
+1 Would like to see us use a single "Reviewed" event.
```suggestion
        SendClientSegmentEventMutation(atmosphere, 'Join Request Reviewed', {
          action: 'Deny'
        })
```